ui/raidboss/data/04-sb/dungeon/doma_castle.txt

```diff
@@ -0,0 +1,137 @@
+1146.7 "2-Tonze Magitek Missile" sync /:Magitek Hexadrone:20A3:/
+1156.8 "Magitek Missiles" sync /:Magitek Hexadrone:20A5:/
+1166.2 "Hexadrone Bits" sync /03:........:Added new combatant Hexadrone Bit/
+1171.7 "Bits Activate"
+1174.7 "2-Tonze Magitek Missile" sync /:Magitek Hexadrone:20A3:/
+
+1181.3 "Circle Of Death" sync /:Magitek Hexadrone:20A2:/ jump 1083.9
+1187.4 "Hexadrone Bits"
+1192.9 "Bits Activate"
+1195.0 "Magitek Missiles"
+1203.8 "2-Tonze Magitek Missile"
+1216.0 "Magitek Missiles"
+
+
+### Hypertuned Grynewaht
+# -ii 20A9 20AB 20AC 20AE 20A7 2447
+2000 "Start" sync /00:0839:The Hall Of The Scarlet Swallow will be sealed off/ window 2000,0
+2009.4 "Chainsaw" duration 5.5 sync /:Hypertuned Grynewaht:20A8:/
+2022.7 "Delay-Action Charge" sync /:Hypertuned Grynewaht:20AD:/
+2023.1 "Gunsaw" duration 5.0  sync /:Hypertuned Grynewaht:20AA:/
+2045.4 "Thermobaric Charge" sync /:Hypertuned Grynewaht:20AF:/
+2048.8 "--sync--" sync /:Hypertuned Grynewaht:20B0:/
+2057.0 "Gunsaw" duration 5.0  sync /:Hypertuned Grynewaht:20AA:/
+2070.3 "Delay-Action Charge/Clean Cut" sync /:Hypertuned Grynewaht:20AD:/
+2071.7 "--sync--" sync /:Hypertuned Grynewaht:20B0:/
+2079.9 "Gunsaw" duration 5.0  sync /:Hypertuned Grynewaht:20AA:/
+2093.2 "Delay-Action Charge/Clean Cut" sync /:Hypertuned Grynewaht:20AD:/
+2097.6 "Hexadrone Bits" sync /03:........:Added new combatant Hexadrone Bit/
+2103.1 "Bits Activate"
+2108.2 "Chainsaw" duration 5.5 sync /:Hypertuned Grynewaht:20A8:/
+2115.9 "Thermobaric Charge" sync /:Hypertuned Grynewaht:20AF:/
+2118.4 "--sync--" sync /:Hypertuned Grynewaht:20B0:/
+2124.7 "Gunsaw" duration 5.0 sync /:Hypertuned Grynewaht:20AA:/
+2138.0 "Delay-Action Charge" sync /:Hypertuned Grynewaht:20AD:/
+2140.0 "Clean Cut" sync /:Magitek Chakram:20B1:/
+2141.4 "--sync--" sync /:Hypertuned Grynewaht:20B0:/
+2149.6 "Gunsaw" duration 5.0 sync /:Hypertuned Grynewaht:20AA:/
+2162.9 "Delay-Action Charge/Clean Cut" sync /:Hypertuned Grynewaht:20AD:/
+2167.4 "--sync--" sync /:Hypertuned Grynewaht:20B0:/
+2168.5 "Hexadrone Bits" sync /03:........:Added new combatant Hexadrone Bit/
+2174.0 "Bits Activate"
+2180.0 "Chainsaw" duration 5.5 sync /:Hypertuned Grynewaht:20A8:/
+2187.7 "Thermobaric Charge" sync /:Hypertuned Grynewaht:20AF:/
+2189.0 "Clean Cut" sync /:Magitek Chakram:20B1:/
+
+2194.2 "--sync--" sync /:Hypertuned Grynewaht:20B0:/
+2202.4 "Gunsaw" duration 5.0 sync /:Hypertuned Grynewaht:20AA:/
+2215.7 "Delay-Action Charge/Clean Cut" sync /:Hypertuned Grynewaht:20AD:/
+2218.2 "Hexadrone Bits" sync /03:........:Added new combatant Hexadrone Bit/
+2223.7 "Bits Activate"
+2228.7 "Chainsaw" duration 5.5 sync /:Hypertuned Grynewaht:20A8:/
+2236.4 "Thermobaric Charge" sync /:Hypertuned Grynewaht:20AF:/
+2242.8 "Chainsaw" duration 5.5 sync /:Hypertuned Grynewaht:20A8:/
+
+2257.0 "--sync--" sync /:Hypertuned Grynewaht:20B0:/
+# Loop begins here, but the jump is attached to the Delay-Action/Clean Cut to preserve the duration timer for Gunsaw
```

Comment:
Ah, thanks for the comment! Sorry about the duration not preserving.  We should fix that too.  ;;